### PR TITLE
add-new-linux-quiz-question

### LINKED ISSUE #16: Adding Linux quiz questions
[developerquiz.org](developerquiz.org) currently has over 600 quiz questions. We are looking to expand on the Linux questions and we encourage other developers to add their quiz questions to the site.

You can find the complete list of questions below.
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/linux-quiz.ts

You can add your own questions to the top of that file.
**Please first check to make sure your question doesn't already exist in the file before creating a PR.**

Here is an example format for the questions. 

```
  {
    Question: "In Linux, what is a maximum length for a filename?",
    Answer: "255 Bytes",
    Distractor1: "225 Bytes",
    Distractor2: "235 Bytes",
    Distractor3: "295 Bytes",
    Explanation:
      "The maximum length for a file name is 255 bytes. The maximum combined length of both the file name and path name is 4096 bytes.",
    Link: "https://www.ibm.com/docs/en/spectrum-protect/8.1.9?topic=parameters-file-specification-syntax",
  },
```
For the `link` field, please make sure to use a freeCodeCamp article, freeCodeCamp YouTube video or official documentation.
If you choose to reference a video, please make sure to include the timestamp for the topic covered.

You can read more about how to create timestamps in this [helpful article](https://www.lifewire.com/link-to-specific-part-of-youtube-video-1616414#). 

This issue will not be assigned to anyone and will remain open for multiple contributors. 
**Please do not assign yourself to this issue or close it.**

Happy contributing!

### PR DESCRIPTION
**Pull request by MazontheMoon**
## Summary of changes

Add 'userdel" command question

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1694,6 +1694,16 @@ const linuxQuiz = [
     Distractor3: "cat /var/log/syslog",
     Explanation: "The cd command stands for change directory and is used to change to a directory in the terminal. The cd command is not used to display logs.",
     Link: "https://www.youtube.com/watch?v=ROjZy1WbCIA&t=3472s"
+  },
+  {
+    Question: "In Linux, when you remove a users account using the 'userdel' command, what happens to the data files stored in that users directory?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1699]:
```suggestion
    Question: "In Linux, when you remove a user's account using the 'userdel' command, what happens to the data files stored in that user's directory?",
```

**Review by jdwilkin4 — COMMENTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/linux-quiz.ts (hunk 1 above)
Comment (MazontheMoon) [reply src/data/linux-quiz.ts]: Thank you for the feedback and I've made the changes as requested 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**